app/main.py

```diff
@@ -1,25 +1,78 @@
+from typing import Type
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: "IntegerRange", name: str) -> None:
+        self.name = "_" + name
+
+    def __get__(self,
+                instance: "SlideLimitationValidator",
+                owner: "IntegerRange"
+                ) -> int:
+        getattr(instance, self.name)
+
+    def __set__(self,
+                instance: "SlideLimitationValidator",
+                value: int
+                ) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(f"{self.name} must be between "
+                             f"{self.min_amount} and {self.max_amount}!")
+        setattr(instance, self.name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one